doc/source/whatsnew/v3.0.0.rst

```diff
@@ -76,6 +76,8 @@ Other enhancements
 - Support passing a :class:`Iterable[Hashable]` input to :meth:`DataFrame.drop_duplicates` (:issue:`59237`)
 - Support reading Stata 102-format (Stata 1) dta files (:issue:`58978`)
 - Support reading Stata 110-format (Stata 7) dta files (:issue:`47176`)
+- Refactor classes in ``pandas.io.sql`` to favor their own implementation of ``execute`` instead of relying on driver's ``execute``.
```

Comment:
Appreciate the note but we should try and make it interesting to non-developers that are just using the library. Maybe we can just combine these into:

```
- Errors occurring during SQL I/O will now throw a generic pandas.DatabaseError instead of the raw Exception type from the underlying driver manager library
- ```

Reply:
Sure thing, makes sense !